packages/inventory-compliance/package.json

```diff
@@ -32,6 +32,12 @@
         "react-content-loader": "^3.4.1",
         "lodash": "4.17.15"
     },
+    "jest": {
```

Comment:
You don't really need to set jest for monorepo in package, instead if you want to run only compliance tests you can run `npm run test packages/inventory-compliance` from root folder.